### PR TITLE
[Mailer] Move CHANGELOG entry

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

This entry belongs to the bridge, not the component

### CODE REVIEW

File: src/Symfony/Component/Mailer/Bridge/Amazon/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3
+---
+
+* Add support for `X-SES-SOURCE-ARN` in `symfony/amazon-mailer`
```
Comment (derrabus) [inline src/Symfony/Component/Mailer/Bridge/Amazon/CHANGELOG.md]:
```suggestion
* Add support for `X-SES-SOURCE-ARN`
```

Comment (OskarStark) [reply src/Symfony/Component/Mailer/Bridge/Amazon/CHANGELOG.md]: Indeed

Comment (OskarStark) [reply src/Symfony/Component/Mailer/Bridge/Amazon/CHANGELOG.md]: Done

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Good catch, thanks @OskarStark.

**Pull request merged**